molgenis-data/src/main/java/org/molgenis/data/meta/MetaDataService.java

```diff
@@ -183,18 +183,16 @@
 	/**
 	 * Adds an Attribute to an EntityMeta
 	 *
-	 * @param entityName
 	 * @param attribute
 	 */
-	void addAttribute(String entityName, AttributeMetaData attribute);
+	void addAttribute(AttributeMetaData attribute);
```

Comment:
How does the metadataservice now know which entity the attribute will be added 2?